charts/linkerd2/README.md

```diff
@@ -0,0 +1,139 @@
+# Linkerd2 Helm Chart
+
+Linkerd is a *service mesh*, designed to give platform-wide observability,
+reliability, and security without requiring configuration or code changes.
+
+Linkerd is a Cloud Native Computing Foundation ([CNCF][cncf]) project.
+
+## Quickstart and documentation
+
+You can run Linkerd on any Kubernetes 1.12+ cluster in a matter of seconds. See
+the [Linkerd Getting Started Guide][getting-started] for how.
+
+For more comprehensive documentation, start with the [Linkerd
+docs][linkerd-docs].
+
+## Prerequisite: identity certificates
+
+The identity component of Linkerd requires setting up a trust anchor
+certificate, and an issuer certificate with its key. These need to be provided
+to Helm by the user (unlike when using the `linkerd install` CLI which can
```

Comment:
Maybe remove the _(unlike when using the `linkerd install` CLI which can generate these automatically)_, since `linkerd install` is irrelevant when the users are reading this README on the Helm repo.

Reply:
For people trying to install Linkerd and stumbling across the chart first for whatever reason, I wanted to give a _hint_ to the alternate installation method, letting them know it'd be easier...